packages/@divvi/mobile/src/transactions/feed/NftFeedItem.tsx

```diff
@@ -33,6 +34,10 @@ function NftFeedItem({ transaction }: Props) {
     })
   }
 
+  if (getAppConfig().experimental?.disableNfts) {
+    return null
+  }
```

Comment:
this is probably not necessary since we no longer fetch nft txs, but there could be an edge case where this is cached.